docs/guide/javaee/jms.adoc

```diff
@@ -1,16 +1,17 @@
 ### JMS
 
-Messaging support is provided through the http://camel.apache.org/jms.html[jms,window=_blank]
+Messaging support is provided through the http://camel.apache.org/jms.html[JMS,window=_blank]
 component which integrates with the {wildfly} Messaging (https://activemq.apache.org/artemis/)[ActiveMQ Artemis,window=_blank]) subsystem.
 
 Integration with other JMS implementations is possible through configuration of vendor specific resource adapters, or if not available, by using the JBoss Generic JMS resource adapter.
 
 
 #### {wildfly} JMS configuration
 
-The {wildfly} https://docs.jboss.org/author/display/WFLY8/Messaging+configuration[messaging subsystem,window=_blank] can be configured from within the standard {wildfly} XML configuration files e.g. standalone.xml.
+You can configure the {wildfly} https://docs.jboss.org/author/display/WFLY8/Messaging+configuration[messaging subsystem,window=_blank] through the standard {wildfly} XML configuration files.
+For example, standalone.xml file.
 
-For the examples that follow we use the embedded ActiveMQ Artemis in memory instance. We first configure a new JMS queue on the messaging subsystem by adding the following XML configuration to the jms-destinations section.
+For the examples, follow the embedded ActiveMQ Artemis in memory instance. You need to configure a new JMS queue on the messaging subsystem, by adding the following XML configuration to the jms-destinations section.
```

Comment:
This rewording doesn't read so well to me. I think it'd be better to keep the original wording and replace 'we' with 'you' like you have done elsewhere.

Reply:
fixed as suggested.